changelog.txt

```diff
@@ -1,5 +1,11 @@
 Version 3.3.0, April 1, 2020
 
+New command-line options:
+  The Nullness Checker now supports the option -Alint=trustArrayLenZero to
+  trust @ArrayLen(0) annotations when determining the type of
+  Collections.toArray. See the new manual section
+  http://checkerframework.org/manual/#nullness-collection-toarray
```

Comment:
I would omit this for brevity.  (Because it should be self-evident to read the manual for any change that is briefly described here.)